src/DssProxyRegistry.sol

```diff
@@ -23,20 +23,15 @@ contract DssProxyRegistry {
     mapping (address => address) public proxies;
     mapping (address => uint256) public isProxy;
 
-    function _salt(address owner_) internal view returns (uint256 salt) {
-        salt = uint256(keccak256(abi.encode(owner_, seed[owner_])));
-    }
-
-    function _code(address owner_) internal pure returns (bytes memory code) {
-        code = abi.encodePacked(type(DssProxy).creationCode, abi.encode(owner_));
-    }
-
     function build(address owner_) external returns (address payable proxy) {
         proxy = payable(proxies[owner_]);
         require(proxy == address(0) || DssProxy(payable(proxy)).owner() != owner_, "DssProxyRegistry/proxy-registered-to-owner"); // Not allow new proxy if the user already has one and remains being the owner
-        seed[owner_]++;
-        uint256 salt = _salt(owner_);
-        bytes memory code = _code(owner_);
+
+        uint256 _seed = seed[owner_];
+        seed[owner_] = ++_seed;
```

Comment:
could this still be `seed[owner_]++`? or is there a reason for preferring this way?

Reply:
We want to increment seed[owner_] but also _seed as well since we use it below in the salt calculation (and we don't want to read from storage again).

Maybe we can do it in one line like this:
`uint256 _seed = ++seed[owner_];`

Reply:
couldn't we do that in line 32 then?

Reply:
Yeah, I'll run it to see there is no gas increase (but believe there shouldn't be).